tests/migration/Core/V6_4/Migration1665267882RenameCountryVatTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Shopware\Tests\Migration\Core\V6_4;
+
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelTestBehaviour;
+use Shopware\Core\Migration\V6_4\Migration1665267882RenameCountryVat;
+
+class Migration1665267882RenameCountryVatTest extends TestCase
+{
+    use KernelTestBehaviour;
```

Comment:
Please use the `MigrationTestTrait`, instead it won't boot the whole kernel for each test execution

You can get the connection then with
```php
$connection = KernelLifecycleManager::getConnection();
```

Reply:
I use the development docker environment.
Previously I could start the test with `vendor/bin/phpunit --configuration platform/phpunit.xml.dist platform/tests/migration/Core/V6_4/Migration1665267882RenameCountryVatTest.php`, but after the change it does not work anymore and I get the error `Trait 'Shopware\Tests\Migration\MigrationTestTrait' not found` (Import and use were set correctly).
Do I need to use another command or options?

Reply:
the new test structure does not work that well with the development setup, as the `/tests` are only registered as `autoload-dev`, therefor the classes can not be autoloaded

Technically there is no need anymore for the development template, you can use platform standalone, but there is sadly no documentation for that right now